io.dbeaver.DBeaverCommunity.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.dbeaver.DBeaverCommunity</id>
+    <launchable type="desktop-id">io.dbeaver.DBeaverCommunity.desktop</launchable>
+    <metadata_license>ALS</metadata_license>
+    <project_license>LicenseRef=https://dbeaver.io/files/dbeaver_license.txt</project_license>
+    <name>DBeaver Community</name>
+    <url type="homepage">https://dbeaver.io</url>
+    <summary>DBeaver Community. Universal Database Tool.</summary>
+    <description>
+        <p>Free multi-platform database tool for developers, SQL programmers, database administrators and analysts. Supports all popular databases: MySQL, PostgreSQL, MariaDB, SQLite, Oracle, DB2, SQL Server, Sybase, MS Access, Teradata, Firebird, Derby, etc.</p>
+        <p>This is the official build of DBeaver, packaged into a Flatpak. This repackaging is not supported by DBeaver Community Edition.</p>
+    </description>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://dbeaver.io/wp-content/uploads/2018/03/data_edit-450x220.png</image>
```

Comment:
Also, I remember I've read somewhere that it's better to have these screenshots in a higher resolution. In their Web they have these (in case you would like to use them instead):

- https://dbeaver.io/product/dbeaver-ss-mock.png
- https://dbeaver.io/product/dbeaver-ss-erd.png
- https://dbeaver.io/product/dbeaver-ss-classic.png
- https://dbeaver.io/product/dbeaver-ss-dark.png